test/system/scenarios/testMarketStopOrders.js

```diff
@@ -0,0 +1,18 @@
+const TestMarketStopOrders = (trader1, trader2, trader3) => {
+  console.log('******************** Scenario 4 Test market stop orders  *********************');
+  return trader1.init_state(trader1.config)
+    .then(() => trader2.init_state(trader2.config))
+    .then(() => trader3.init_state(trader3.config))
+    .then(() => trader1.placeLimitOrderSpread('B', 0.21, 1, 3, 0.01))
+    .then(() => trader1.placeLimitOrderSpread('S', 0.25, 1, 3, 0.01))
+    .then(() => trader2.createOrder((trader2.getLowestSellPrice(trader1.config, 'S') + 0.05), 'B',  0.25, 'S', 0.25))
```

Comment:
No need to calculate the price - this scenario wants trader2 to place a Stop Buy order with a value of 0.25 and a stop of 0.265